lib/merchant_repository.rb

```diff
@@ -0,0 +1,56 @@
+require 'CSV'
+# require './lib/merchant'
+
+class MerchantRepository
+  attr_reader :all_merchants
+
+  def initialize(path)
+    @all_merchants = create_merchants(path)
+  end
+
+  def create_merchants(path)
+  @merchants = CSV.foreach(merchants, headers: true, header_converters: :symbol).map do |merchant|
```

Comment:
I think `@merchants` can just be `merchants` here and not an instance variable, since the output is assigned to `@all_merchants`. 

Also, I think the `merchants` in `foreach(merchants` should be `path`, since that is what we are bringing into this method to iterate through.